D-lama-service/Data/ProjectEntities/TextDataPoint.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Data.ProjectEntities
+{
+    /// <summary>
+    /// Text data point entity.
+    /// </summary>
+    public class TextDataPoint : DataPoint
+    {
+        [Required]
+        public string Content { get; set; }
+
+        // Required foreign key property
+        public int ProjectId { get; set; }
+        // Required reference navigation to principal
+        [JsonIgnore]
+        public Project Project { get; set; } = null!;
+        // Optional reference navigation to principal
+        public User? Labeler { get; set; }
+        public Label? Label { get; set; }
```

Comment:
I had a though about that again and correct me if I am wrong, but all we need for setting a label to a datapoint is:
- the id of the datapoint
- the id of the label
- the id of the labeler
One thing we do not need is the content and thinking about how big it can get, we should avoid having to duplicate it. Or am I missing something?

Reply:
this is the notation for an **optional** reference navigation to a principal as I found it here: https://learn.microsoft.com/en-us/ef/core/modeling/relationships/one-to-many in the section "Optional one-to-many". But If I see the table in the db, yes I am not sure if it is correct like this, will say, I do not understand why we do not have the id but the whole reference. I will delete these attributes and the related collections in the User und Label Entities, as it is something for the issue about the labelling process.

Reply:
Nevermind, I just realised that it correctly just shows the LabelId and LabellerId, so it was as I had intended it. However, I already deleted the lines. It is better if it will be covered by the labelling process branch.

Reply:
I think so too. Thank you!